packages/claims/src/claims/claims.ts

```diff
@@ -54,10 +58,13 @@ export class Claims implements IClaims {
     if (this.keys?.publicKey) {
       return this.keys.publicKey;
     }
-    const hash = await ethers.utils.keccak256(await this.signer.getAddress());
-    const sig = await this.signer.signMessage(ethers.utils.arrayify(hash));
+    const hash = keccak256(await this.signer.getAddress());
+    const sig = await this.signer.signMessage(arrayify(hash));
+    const digest = arrayify(hashMessage(arrayify(hash)));
     // eslint-disable-next-line no-return-assign
-    return this.keys.publicKey = Promise.resolve(ethers.utils.recoverPublicKey(hash, sig));
+    const publicKey = `02${recoverPublicKey(digest, sig).slice(4, 68)}`;
```

Comment:
why is this '02' prefixed?

Reply:
Its prefix used in public keys. For uncompressed key the prefix is 04, for compressed are - 02, 03